parlai/scripts/train_model.py

```diff
@@ -5,26 +5,24 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree. An additional grant
 # of patent rights can be found in the PATENTS file in the same directory.
-"""Train a model.
+"""The standard way to train a model. After training, also computes validation
+and test error.
 
-After training, computes validation and test error.
+The user must provide a model (with ``--model``) and a task (with ``--task``).
```

Comment:
Note: a user can also provide a task to be used with `PytorchDataTeacher` via `-pyt`, or can alternatively provide a pytorch `Dataset` via `-pytd`